sdk/compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs

```diff
@@ -330,6 +331,7 @@ getIntegrationTests registerTODO scenarioService (packageDbPath, packageFlags) =
                 , optPackageImports = packageFlags
                 , optDetailLevel = PrettyLevel (-1)
                 , optEnableInterfaces = EnableInterfaces True
+                , optUpgradeInfo = defaultUpgradeInfo { uiWarnBadInterfaceInstances = Just True }
```

Comment:
TODO: properly recursively change this field, instead of using defaultUpgradeInfo